packages/Avalonia/AvaloniaBuildTasks.targets

```diff
@@ -74,7 +74,7 @@
       ReportImportance="$(AvaloniaXamlReportImportance)"/>
     <Exec 
       Condition="'$(_AvaloniaUseExternalMSBuild)' == 'true'"
-      Command="dotnet msbuild /nodereuse:false $(MSBuildProjectFile) /t:GenerateAvaloniaResources /p:_AvaloniaForceInternalMSBuild=true /p:Configuration=$(Configuration) /p:TargetFramework=$(TargetFramework) /p:BuildProjectReferences=false"/>
+      Command="dotnet msbuild /nodereuse:false $(MSBuildProjectFile) /t:GenerateAvaloniaResources /p:_AvaloniaForceInternalMSBuild=true /p:Configuration=$(Configuration) /p:TargetFramework=$(TargetFramework) /p:RuntimeIdentifier=$(RuntimeIdentifier) /p:BuildProjectReferences=false"/>
```

Comment:
This was causing compile errors with if I change xaml project target framework to "net7.0".
Main msbuild task was running with a runtime identifier set (by default in net7.0), while external msbuild task wasn't (no idea why, haven't checked too deep). 
As the result, IntermediateOutputPath was different between these two runs, and resource files were missing.